src/gui/mainwindow.cpp

```diff
@@ -1443,6 +1443,7 @@ void MainWindow::loadPreferences()
     }
 
     showStatusBar(pref->isStatusbarDisplayed());
+    m_statusBar->showStatusBarExternalIP(pref->isStatusbarExternalIPDisplayed());
```

Comment:
This should be removed. Status bar itself should connect to Preferences::changed signal.